### ISSUE TITLE
Is this project maintained anymore?

### ISSUE THREAD

**Issue by brhx** [typekit/webfontloader#411]
There's hasn't been a single commit on master in over a year and a half; I wanted to check whether or not it's worth my team or I using or contributing to this project if it's no longer maintained. There are many open issues and PR's with no recent activity, from what I can tell.

No worries if not, of course. Just wanted to make sure.

**Comment by DanielRuf** [comment]
Would like to know this too.

**Comment by AdelDima** [comment]
Me too

**Comment by jasonfb** [comment]
I am here wondering if this can/will be ported to ES6 syntax, with a module that can be loaded using webpack. It seems like a re-write would be a good call.

**Comment by DanielRuf** [comment]
Well, the last commit was a few years ago - so probably no.

**Comment by jasonfb** [comment]
Let me rephrase, SHOULD this project be ported to an ES6 module ?

**Comment by jasonfb** [comment]
alternatively, does anyone know of an ES6 module that would be a good replacement?

**Comment by micobarac** [comment]
[Font Face Observer](https://github.com/bramstein/fontfaceobserver)

**Comment by adamreisnz** [comment]
Font Face Observer has been last updated 2.5 years ago, so I wouldn't cheer too soon 😆

**Comment by notboring** [comment]
Landed here since Webfontloader isn't really working anymore on Chrome. Webfontloader has been last updated seven years ago while Font Face Observer has been updated two weeks ago.

Seems Webfontloader is not maintained anymore and Font Face Observer is an up to date alternative.
